### PR TITLE
Docs: Convert "getting started - tutorial" from RST to MD

### PR DESCRIPTION
**Pull request by tbrlpld**
Partially replaces #6834 

Compare locally with
```sh
git diff master:./docs/getting_started/tutorial.rst ./docs/getting_started/tutorial.md
```

### CODE REVIEW

File: docs/getting_started/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,867 @@
...
+This tutorial uses [venv](https://docs.python.org/3/tutorial/venv.html), which is packaged with Python 3.
+
+**On Windows** (cmd.exe):
+
+```bat
+$ python3 -m venv mysite\env
+$ mysite\env\Scripts\activate.bat
+```
+
+**On Unix or MacOS** (bash):
```
Comment (kaedroho) [inline docs/getting_started/tutorial.md:42]: What about Linux? Mac OS is closer to Unix than Linux!

File: docs/getting_started/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+```eval_rst
+.. important::
+   Before installing Wagtail, it is necessary to install the **libjpeg** and **zlib** libraries, which provide support for working with JPEG, PNG and GIF images (via the Python **Pillow** library).
+   The way to do this varies by platform—see Pillow's
+   `platform-specific installation instructions <https://pillow.readthedocs.org/en/latest/installation.html#external-libraries>`_.
+```
+
+### Create and activate a virtual environment
+
+We recommend using a virtual environment, which provides an isolated Python environment.
```
Comment (kaedroho) [inline docs/getting_started/tutorial.md:32]: I think for new Python developers "which provides an isolated Python environment" might not be very clear. How about "which isolates installed dependencies from other projects"?

File: docs/getting_started/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,867 @@
...
+```
+
+**On Unix or MacOS** (bash):
+
+```sh
+$ python3 -m venv mysite/env
+$ source mysite/env/bin/activate
+```
+
+**For other shells** see the [venv documentation](https://docs.python.org/3/library/venv.html).
```
Comment (kaedroho) [inline docs/getting_started/tutorial.md:49]: venv, should be surrounded by backticks

File: docs/getting_started/tutorial.md (hunk 1 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md:42]: I tried to not make any editorial changes. This was the phrasing as before.

File: docs/getting_started/tutorial.md (hunk 2 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md:32]: Sounds good to me.

File: docs/getting_started/tutorial.md (hunk 1 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md]: Changed in 442eff039d27938ad0450c8c785039be6772b9e1

File: docs/getting_started/tutorial.md (hunk 2 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md]: Changed in 3ce150c16dd3ffc4ecc0b124119a4cd2c672376a

File: docs/getting_started/tutorial.md (hunk 3 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md]: Added in 0c4d82ac3c99e688c02e51d321cf8b4669e6c486

File: docs/getting_started/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,867 @@
...
+
+## Install and run Wagtail
+
+### Install dependencies
+
+Wagtail supports Python 3.6, 3.7, 3.8 and 3.9.
+
+To check whether you have an appropriate version of Python 3:
+
+```sh
```
Comment (kaedroho) [inline docs/getting_started/tutorial.md:17]: Doesn't seem to be any syntax highlighting coming through here, should this be ``bash``?

File: docs/getting_started/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,867 @@
...
+
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-homepage{% endblock %}
+
+{% block content %}
+    {{ page.body|richtext }}
+{% endblock %}
+
+``base.html`` refers to a parent template and must always be the first template tag used in a template. Extending from this template saves you from rewriting code and allows pages across your app to share a similar frame (by using block tags in the child template, you are able to override specific content within the parent template).
```
Comment (kaedroho) [inline docs/getting_started/tutorial.md:184]: Should these descriptions be below the `` ``` ``?

**Comment by kaedroho** [comment]
Just a couple of small tweaks, then I think this is ready for merge!

File: docs/getting_started/tutorial.md (hunk 4 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md:17]: I just tried, neither is showing any particular highlighting on the line. 

... Ooh, it's actually still `console` that works. I had never seen that before.

Comment (tbrlpld) [reply docs/getting_started/tutorial.md:17]: Fixed in a4551e996aba65c960316c30d532f034da03ed7e

File: docs/getting_started/tutorial.md (hunk 5 above)
Comment (tbrlpld) [reply docs/getting_started/tutorial.md]: Yup. Good catch. Thank you. 

Fixed in b0229fe52311d3cb240bd12233b6c540522c0a0e

**Review by kaedroho — APPROVED** [review]
Looks good, thanks!

**Pull request merged**